File: codeforces/1804/1804A Lame King.java
import java.lang.Math;
import java.util.Scanner;

public class LameKing {
	static Scanner sc = new Scanner(System.in);

	public static void main(String args[]) {
		int t = sc.nextInt();
		while (t-- > 0) {
			int a = sc.nextInt(), b = sc.nextInt();
			a = Math.abs(a);
			b = Math.abs(b);
			if (a < b) {
				a += b;
				b = a - b;
				a = a - b;
			}
			System.out.println(a + b + Math.max(0, a - b - 1));
		}
	}
}

File: codeforces/1804/1804A Lame King.py
t = int(input())
while t:
    t -= 1
    lst = list(map(int, input().split()))
    a = lst[0]
    b = lst[1]
    a = abs(a)
    b = abs(b)
    if a < b:
        a, b = b, a
    print(a + b + max(0, a - b - 1))
